services/wallet/common/log_parser.go

```diff
@@ -20,20 +20,24 @@ type EventType string
 
 const (
 	// Transaction types
-	EthTransfer        Type = "eth"
-	Erc20Transfer      Type = "erc20"
-	Erc721Transfer     Type = "erc721"
-	UniswapV2Swap      Type = "uniswapV2Swap"
-	UniswapV3Swap      Type = "uniswapV3Swap"
-	HopBridgeFrom      Type = "HopBridgeFrom"
-	HopBridgeTo        Type = "HopBridgeTo"
-	unknownTransaction Type = "unknown"
+	EthTransfer           Type = "eth"
+	Erc20Transfer         Type = "erc20"
+	Erc721Transfer        Type = "erc721"
+	Erc1155SingleTransfer Type = "erc1155"
+	Erc1155BatchTransfer  Type = "erc1155"
```

Comment:
I understand these identify transfer types in the `transactions` table. Using the exact string for single and batched will lose this information that we have at runtime. Is this intended? Won't it be helpful to propagate the runtime info to DB by using different identifiers (e.g. `erc1155single` and `erc1155batch`)?